packages/cli-hydrogen/src/cli/services/preview.ts

```diff
@@ -53,6 +62,25 @@ export async function previewInWorker({directory, port}: PreviewOptions) {
     }
   }
 
+  function parseEnvPath(envPath: string): EnvConfig {
```

Comment:
Have you considered using the `parse` function from the `envfile` package? We use it in `cli-kit` and should be available to you as well.

Reply:
Awesome! 👍